engine/src/main/java/org/terasology/logic/console/commands/ServerCommands.java

```diff
@@ -151,6 +152,11 @@ public void save() {
         storageManager.requestSaving();
     }
 
+    @Command(shortDescription = "Deletes the specified chunk and re-generate it", runOnServer = true)
+    public void purgeChunk(@CommandParam("x") int x, @CommandParam("y") int y, @CommandParam("z") int z) {
+        chunkProvider.purgeChunk(new Vector3i(x, y, z));
```

Comment:
Suggestion for future commit: Make this print something to the console, maybe the boolean value returned to tell the player if the purge did something or not?